sdk/src/types/positions/utils.rs

```diff
@@ -55,6 +55,37 @@ pub fn mul_div(a: U256, b: U256, denominator: U256) -> U256 {
     U256::from(quotient)
 }
 
+pub fn compress_tick(tick: I24, tick_spacing: I24) -> I24 {
+    tick / tick_spacing - if tick % tick_spacing < I24::ZERO { I24::ONE } else { I24::ZERO }
```

Comment:
Just checking that you want `tick/tick_spacing -1:0` and not `tick/ (tick_spacing- 1:0)`